source/utils/generateCommitMessage.js

```
@@ -0,0 +1,61 @@
+			case 'gpt-4o-mini':
+				response = await openAiModel(model, flags, gitDiffContent);
+				break;
+			case 'llama3.1:8b':
+				response = await ollamaModel(model, flags, gitDiffContent);
+				break;
+			default:
+				throw new Error('Unsupported model selected');
+		}
+		console.log('response', response);
+
+		if (response && response.category && response.message) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)